packages/components/src/components/notification/notification.stories.ts

```diff
@@ -0,0 +1,357 @@
+          });
+
+          notification.style.width = '251px';
+
+          document.body.append(notification);
+          return notification.toast();
+        }
+
+        buttons.forEach(button => {
+          button.addEventListener('click', () => {
+            notifyBottomCenter(button.getAttribute('notification-type'));
+          });
+        });
+      </script>
+    `;
+  },
+  play: async ({ canvasElement }: { canvasElement: HTMLUnknownElement }) => {
+    const button = canvasElement.querySelector('#bottom-center');
+    await userEvent.click(button!);
+  }
+};
+
+/**
+ * Use the `base`, `icon`, `content`, `message`, `duration-indicator-current`, `duration-indicator-total` and `close-button`, part selectors to customize the notification.
+ */
+
+export const Parts = {
+  parameters: {
+    controls: {
+      exclude: [
+        'base',
+        'icon',
+        'content',
+        'message',
+        'duration-indicator-current',
+        'duration-indicator-total',
+        'close-button'
+      ]
+    }
+  },
+  render: (args: any) => {
+    return generateTemplate({
+      axis: {
+        y: {
+          type: 'template',
+          name: 'sd-notification::part(...){outline: solid 2px red}',
+          values: [
+            'base',
+            'icon',
+            'content',
+            'message',
+            'duration-indicator-current',
+            'duration-indicator-total',
+            'close-button'
+          ].map(part => {
+            return {
+              title: part,
+              value: `<style>#part-${part} sd-notification::part(${part}){border: solid 2px red; z-index: 2;}</style><div id="part-${part}">%TEMPLATE%</div>`
+            };
+          })
+        }
+      },
+      args,
+      constants: [
+        { type: 'attribute', name: 'duration', value: Infinity },
+        { type: 'attribute', name: 'duration-indicator', value: true },
+        { type: 'attribute', name: 'closable', value: true },
+        { type: 'attribute', name: 'open', value: true }
+      ]
+    });
+  }
+};
+
+/**
+ * sd-notifications are fully accessibile via keyboard.
+ */
+
+export const Mouseless = {
+  render: (args: any) => {
+    // eslint-disable-next-line lit-a11y/no-autofocus
+    return html`<div class="mouseless" autofocus>
+      ${generateTemplate({
+        args,
+        constants: [
+          { type: 'attribute', name: 'closable', value: true },
+          { type: 'attribute', name: 'open', value: true }
+        ]
+      })}
+    </div>`;
+  },
+
+  play: async ({ canvasElement }: { canvasElement: HTMLUnknownElement }) => {
+    const el = canvasElement.querySelector('.mouseless sd-notification');
+    await waitUntil(() => el?.shadowRoot?.querySelector('sd-button'));
+    console.log('before trying to focus');
+    console.log(document.activeElement);
+
+    await userEvent.tab();
+    // await clickOnElement(canvasElement.querySelector('.mouseless sd-notification')!);
```

Comment:
Here are some comment that could be removed. :)

Reply:
Still working on that story :)